.readthedocs.yml

```diff
@@ -1,10 +1,11 @@
 version: 2
 
 build:
-  image: latest
+  os: ubuntu-22.04
```

Comment:
Why this change?

Reply:
Initially, this was a test because readthedocs failed to compile the documentation. I solved the problem in another PR, but it was the same changes.